src/demo/view-configs/plugin-view-type.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable react/button-has-type */
```

Comment:
Would it make sense to add this (and/or the other demos) to cypress? Could be an easy way to make sure things don't break, like we did for the various loaders.

Reply:
Done in https://github.com/vitessce/vitessce/pull/1212/commits/e6570902ddb293750a1693ceba1f157da67f1549 and https://github.com/vitessce/vitessce/pull/1212/commits/f475416edc5aa759dc856a0b2e2a3332f9a368d4